### PR TITLE
[kjobctl] Fix create interactive command.

### PR DESCRIPTION
**Pull request by mbobrovskyi**
#### What type of PR is this?
/kind bug

#### What this PR does / why we need it:
Fix create interactive command on kjobctl. 

#### Which issue(s) this PR fixes:

Fixes #

#### Special notes for your reviewer:
It is not working well after creation, because we don't have object name that generated on server side. We need to use created object instead generated.

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: cmd/experimental/kjobctl/pkg/cmd/create/create.go [hunk 1]
```diff
@@ -588,15 +588,15 @@ func (o *CreateOptions) Run(ctx context.Context, clientGetter util.ClientGetter,
 		return err
 	}
 
-	for _, obj := range objs {
+	for i := range objs {
```
Comment (alculquicondor) [inline cmd/experimental/kjobctl/pkg/cmd/create/create.go:591]: tests that would catch the problem?

Comment (mbobrovskyi) [reply cmd/experimental/kjobctl/pkg/cmd/create/create.go:591]: Added tests to cover it.

File: cmd/experimental/kjobctl/pkg/cmd/create/create_test.go [hunk 2]
```diff
@@ -1123,94 +1128,144 @@ error_path=$(unmask_filename "$SBATCH_ERROR")
...
+
 	testCases := map[string]struct {
-		podName string
-		options *CreateOptions
-		pods    []runtime.Object
-		wantErr string
+		options                *CreateOptions
+		k8sObjs                []runtime.Object
+		kjobctlObjs            []runtime.Object
+		updatePodAfterCreation func(pod *corev1.Pod)
```
Comment (alculquicondor) [inline cmd/experimental/kjobctl/pkg/cmd/create/create_test.go:1139]:
```suggestion
		createMutation func(pod *corev1.Pod)
```

Comment (mbobrovskyi) [reply cmd/experimental/kjobctl/pkg/cmd/create/create_test.go]: Fixed. Thanks!

**Pull request merged**